### DISCUSSION TITLE
"Compare" not présent in drop down list

### DISCUSSION THREAD

**Discussion by spectable** [plausible/analytics#3139] [Self-Hosted Support]
### Past Issues Searched

- [X] I have searched open and closed issues to make sure that the bug has not yet been reported

### Issue is a Bug Report

- [X] This is a bug report and not a feature request, nor asking for self-hosted support

### Using official Plausible Cloud hosting or self-hosting?

Self-hosting

### Describe the bug

The Compare item not present in drop down list. So can't use this functionality
Installer 1 week ago
Just did an docker-compose pull plausible.

Thanks !

### Expected behavior

I should see the Compare item.

### Screenshots

![Capture d’écran_2023-07-13_12-17-07](https://github.com/plausible/analytics/assets/98405/d9852beb-a21f-492f-9e11-5a25999d0eb1)

### Environment

```markdown
- OS:Ubuntu
- Browser: Chrome / Firefox
- Browser Version:  114.0.5735.198  / 115.0.2 (64 bits)
```

**Comment by ruslandoga** [comment]
👋 @spectable 

What version are you using? You can probably find it in `docker-compose.yml`.

**Reply by ruslandoga** [reply]
Note that period comparisons have been added in `v2.0.0` released yesterday: https://github.com/plausible/analytics/discussions/3132

**Comment by spectable** [comment]
After :
cd /opt/plausible
sudo docker compose down --remove-orphans
sudo docker compose pull plausible
sudo docker compose up -d

Here my docker-compose.yml

Shure i missed a step.  Thanks a lot !!!

version: "3.3"
services:
  mail:
    image: bytemark/smtp
    restart: always

  plausible_db:
    # supported versions are 12, 13, and 14
    image: postgres:14-alpine
    restart: always
    volumes:
      - db-data:/var/lib/postgresql/data
    environment:
      - POSTGRES_PASSWORD=postgres

  plausible_events_db:
    image: clickhouse/clickhouse-server:22.8.13.20-alpine
    restart: always
    volumes:
      - event-data:/var/lib/clickhouse
      - ./clickhouse/clickhouse-config.xml:/etc/clickhouse-server/config.d/logging.xml:ro
      - ./clickhouse/clickhouse-user-config.xml:/etc/clickhouse-server/users.d/logging.xml:ro
    ulimits:
      nofile:
        soft: 262144
        hard: 262144

  plausible:
    image: plausible/analytics:v1.5
    restart: always
    command: sh -c "sleep 10 && /entrypoint.sh db createdb && /entrypoint.sh db migrate && /entrypoint.sh run"
    depends_on:
      - plausible_db
      - plausible_events_db
      - mail
    ports:
      - 127.0.0.1:8000:8000
    env_file:
      - plausible-conf.env

volumes:
  db-data:
    driver: local
  event-data:
    driver: local
  geoip:
    driver: local

**Comment by gravelld** [comment]
I'm seeing the same thing in the hosted version. All of my properties except one (blisshq.com) shows the "Compare" option.

I'm not sure if this is related, but that property also has imported Google Analytics data. However, clicking that icon to remove that data still doesn't show the _Compare_ option.

**Reply by metmarkosaric** [reply]
hi @gravelld! If this is about our service on plausible.io, please do [contact customer support](https://plausible.io/contact) as that's the best place for any issues

**Reply by gravelld** [reply]
I didn't want to open more than one discussion - [I saw this was originally posted in the Issues section](https://github.com/plausible/analytics/issues/3138) and it was moved, thus I concluded it wasn't deemed an "issue"...

**Reply by metmarkosaric** [reply]
comparisons are available on all dashboards in the cloud. this thread that you've replied to is about self-hosting. if you're using our cloud product and have any problems there, please do contact our support directly via email so we can help
